browser/src/control/Toolbar.js

```diff
@@ -720,6 +722,16 @@ L.Map.include({
 		}
 	},
 
+	showSnackbar: function() {
```

Comment:
please rename to something describing more what it does :)
like showHasBeenCopiedSnackbar